security_check/main.py

```diff
@@ -36,11 +36,20 @@ def _check_image(image, severity):
     parsed = _run_gcloud(['describe', full_name])
 
     unpatched = 0
-    for vuln in parsed.get('vulz_analysis', {}).get('FixesAvailable', []):
+    for vuln in parsed.get('vulz_analysis', []):
```

Comment:
This has to do with the format returned by gcloud changing. vulz_analysis was initially a dict, but in order to parse the base_image_url below in a generic way, I needed to upgrade the version of gcloud we use, and that means that vulz_analysis is now an array.